pkgs/tools/backup/bacula/default.nix

```
@@ -1,4 +1,6 @@
-{ stdenv, fetchurl, sqlite, postgresql, zlib, acl, ncurses, openssl, readline }:
+{ stdenv, fetchurl, sqlite, postgresql, zlib, acl, ncurses, openssl, readline
+, darwin
```

We usually pass individual frameworks rather than whole `darwin` scope:
```suggestion
, CoreFoundation, IOKit
```